__test__/itemCounter.test.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import itemCounters from '../src/updatedItemCounter.js';
+
+describe('itemCounters', () => {
```

Comment:
- [ ] Since we discussed refactoring the counter function, the test should cover the function soon to be created. Please make sure you are covering all possible edge cases. One of them is when there are no foods displayed on the screen the counter function should return 0. ✍️